server/methods.js

```diff
@@ -0,0 +1,91 @@
+Meteor.methods({
+    "message.build" (params) {
+        return new Chatter.Message({
```

Comment:
Use [check](http://docs.meteor.com/#/full/check) package to validate params! all params in methods should have checks.